lib/Sema/DerivedConformanceDifferentiable.cpp

```diff
@@ -58,24 +58,14 @@ static ValueDecl *getProtocolRequirement(ProtocolDecl *proto, Identifier name) {
 }
 
 // Get the stored properties of a nominal type that are relevant for
-// differentiation.
-// - If the nominal conforms to `Parameterized`, return only the stored
-//   properties marked with `@TFParameter`.
-// - Otherwise, return all stored properties.
-static SmallVector<VarDecl *, 4>
-getStoredPropertiesForDifferentiation(NominalTypeDecl *nominal) {
-  auto &C = nominal->getASTContext();
-  auto *parameterizedProto = C.getProtocol(KnownProtocolKind::Parameterized);
-  SmallVector<VarDecl *, 4> storedProperties;
-  if (TypeChecker::conformsToProtocol(
-          nominal->getDeclaredInterfaceType(), parameterizedProto,
-          nominal->getDeclContext(), ConformanceCheckFlags::Used)) {
-    nominal->getAllTFParameters(storedProperties);
-  } else {
-    storedProperties.append(nominal->getStoredProperties().begin(),
-                            nominal->getStoredProperties().end());
+// differentiation, except the ones tagged `@noDerivative`.
```

Comment:
Very awesome to remove the `Parameterized`-related logic in `Differentiable` derived conformances!